src/Selenium2Driver.php

```diff
@@ -579,7 +579,7 @@ public function getValue($xpath)
     {
         $element = $this->findElement($xpath);
         $elementName = strtolower($element->name());
-        $elementType = strtolower($element->attribute('type'));
+        $elementType = strtolower($element->attribute('type') ?: '');
```

Comment:
As we're dealing with nulls I think doing ```?? ''``` is probably better

Reply:
Lol no - the minimum PHP in compose.json is 5.4!

Reply:
Your code already deals with NULL, so no changes needed to this line I guess.